utils/utils.c

```diff
@@ -278,24 +280,43 @@ void fetchGlfsVolServerFromEnv()
   LOG("mgmt", GB_LOG_INFO, "Block Hosting Volfile Server Set to: %s", gbConf.volServer);
 }
 
-
-int
-initLogging(void)
+static int
+initLogDirAndFiles(char *newLogDir)
```

Comment:
@lxbsz 
Where is that "CRIT: logDir now is /var/log/gluster-block' message :-)

Reply:
Will add this :-)
Thanks.